release_foreman_rpms

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+if [ -z "$1" ];then
+  echo "Pass version to release, e.g. ./release_foreman_rpms 2.2"
+  exit 1
+fi
+
+VERSION=$1 ./download_rpms
+VERSION=$1 ./sign_rpms
+VERSION=$1 ./upload_rpm_signatures
+VERSION=$1 ./upload_rpms
+VERSION=$1 ./process_rpms
+
+# Run the same steps for client but avoid recursion
```

Comment:
Well, you can now also call it for Katello and that doesn't imply a client release. You can also run it just for a client release.